blocks/flag.ts

```diff
@@ -64,7 +65,7 @@ const flagBlock: Block<BlockModule<FlagFunc>> = {
   >(func: {
     default: FlagFunc<TConfig>;
   }) =>
-  ($live: TConfig, { request }: HttpContext) => {
+  async ($live: TConfig, { request }: HttpContext) => {
```

Comment:
this will impact tracings as only async functions are included on tracing. This shouldn't be though but we should keep an eye on it.